src/main.c

```diff
@@ -1,12 +1,20 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+
 #include "includes/sutils.h"
 #include "includes/sinstructions.h"
 #include "includes/svm.h"
 #include "includes/smemory.h"
 #include "includes/stable_utils.h"
 
+#define REPL_HELP \
+  "Usage: senegal [flags] | [senegal-file]\n\n" \
+  "Global options:\n" \
+  "-h, --help                 Print this usage information.\n" \
+  "    --version              Print the Senegal version.\n" \
+  "    --execute [code]       Executes the code provided as an argument with type of a string."
```

Comment:
This is redundant as the repl itself already does it

Reply:
—execute command?

Reply:
Yes

Reply:
Ah ok

Reply:
I’ll fix that 👍🏼